lib/auth/auth.go

```diff
@@ -1269,6 +1269,58 @@ func certRequestDeviceExtensions(ext tlsca.DeviceExtensions) certRequestOption {
 	}
 }
 
+func (a *Server) GenerateOpenSSHCert(ctx context.Context, req *proto.OpenSSHCertRequest) (*proto.OpenSSHCert, error) {
+	if req.Username == "" {
+		return nil, trace.BadParameter("username is empty")
+	}
+	if len(req.PublicKey) == 0 {
+		return nil, trace.BadParameter("public key is empty")
+	}
+	if req.TTL == 0 {
+		return nil, trace.BadParameter("TTL is not set")
+	}
+	if req.TTL < 0 {
+		return nil, trace.BadParameter("TTL must be positive")
+	}
```

Comment:
```suggestion
	if req.TTL <= 0 {
		return nil, trace.BadParameter("TTL must be a positive value")
	}
```

Reply:
I don't see how this helps as I'm already checking if `req.TTL` is `0` above.